client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -0,0 +1,140 @@
+  const classes = useStyles();
+  const options = ['Male', 'Female', 'Other'];
+
+  return (
+    <Box>
+      <Formik
+        initialValues={{
+          ...initial_form_state,
+        }}
+        validationSchema={form_validation}
+        onSubmit={(values) => {
+          console.log(values);
+        }}
+      >
+        <Form>
+          <Grid container alignItems="center" spacing={2}>
+            <Grid item xs={12}>
+              <Typography variant="h6" align="center">
+                Edit Profile
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                First Name
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <Textfield name="firstName" label={<Typography className={classes.label}>firstName</Typography>} />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Last Name
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <Textfield name="lastName" label={<Typography className={classes.label}>lastName</Typography>} />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Gender
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <Select
+                name="gender"
+                options={options}
+                label={<Typography className={classes.label}>gender</Typography>}
+              />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Birth Date
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <DateTimePicker name="birthDate" label={<Typography className={classes.label}>birthDate</Typography>} />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Email Address
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <Textfield name="email" label={<Typography className={classes.label}>email</Typography>} />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Phone Number
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <Textfield name="phone" label={<Typography className={classes.label}>phone</Typography>} />
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                Where You Live
```

Comment:
Maybe a label similar to "address" is less... forward? 😅

Reply:
> 

@ugur-akin could you please explain more what you meant because I seem not to understand you

Reply:
Ah! I just thought "where you live" sounds a bit more private, personal. Some of the other alternatives such as "Address" seems a bit less personal, more "professional" I guess 🤔